File: qz/u2/sisalema/Project in pairs/Student.js
export class Student{
    constructor(id,name,lastName, email, age) {
        this.id = id;
        this.name = name;
        this.lastName = lastName;
        this.email = email;
        this.age = age;
    }
    getId(){
        this.id = id;
    }
    setId(newId) {
        this.id = newId;
    }
    getName() {
        return this.name;
    }
    setName(newName) {
        this.name = newName;
    }
    getLastName() {
        return this.name;
    }
    setLastName(newLastName) {
        this.lastName = newLastName;
    }
}

File: exams/u2/padilla/Padilla 61-70/Padilla 61-70/adrizzx/Q61-70/StudentManagement/src/main/java/ec/edu/espe/studentmanagement/model/Student.java

package ec.edu.espe.studentmanagement.model;

/**
 *
 * @author Adrian Padilla
 */
public class Student {
    private String id;
    private String name;
    private int age;
    private int grade;

    public Student(String id, String name, int age, int grade) {
        this.id = id;
        this.name = name;
        this.age = age;
        this.grade = grade;
    }

    public double computeGradeRatio() {
        return (double) grade / age; // Ejemplo de una regla de negocio
    }

    @Override
    public String toString() {
        return "Student{" + "id=" + id + ", name=" + name + ", age=" + age + ", grade=" + grade + '}';
    }

    // Getters y Setters
    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getAge() {
        return age;
    }

    public void setAge(int age) {
        this.age = age;
    }

    public int getGrade() {
        return grade;
    }

    public void setGrade(int grade) {
        this.grade = grade;
    }
}